ci/upload_to_owncloud.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" && pwd)"
+rootdir=$(realpath "$scriptdir/../")
+
+# shellcheck source=functions.sh
+. "$rootdir/tools/functions.sh"
+
+usage() {
+    echo "usage: $(basename "$0") [-hv] <user> [user] <password> [password] <remote-path> [path] <local-path> [path]"
+    echo "  options:"
+    echo "      -h|--help - display this help."
+    echo "      -v|--verbose - enable verbosity"
+    echo "      -u|--url - webDAV URL (default https://ftp.essensium.com/owncloud/remote.php/dav/files/USERNAME)"
+    echo
+    echo "  positional arguments:"
+    echo "      user - owncloud user name to use for uploading file[s]"
+    echo "      password - owncloud password"
+    echo "      remote-path - path in the cloud, relative to the user home to upload"
+    echo "      local-path - local file or folder to upload"
+    echo ""
+}
+
+main() {
+    if ! OPTS=$(getopt -o 'hvu:' --long help,verbose,url: -n 'parse-options' -- "$@"); then
+        echo echo "Failed parsing options." >&2; usage; exit 1
+    fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -h | --help)            usage; exit 0;;
+            -v | --verbose)         export VERBOSE=true; REDIRECT=">/dev/null 2>&1"; shift;;
```

Comment:
Shouldn't this be
```suggestion
            -v | --verbose)         export VERBOSE=true; REDIRECT=""; shift;;
```

and set REDIRECT as a default?